universal-login-wallet/webpack.config.js

```diff
@@ -61,7 +62,11 @@ module.exports = {
       'process.env.RELAYER_URL': JSON.stringify(process.env.RELAYER_URL),
       'process.env.ENS_DOMAIN_1': JSON.stringify(process.env.ENS_DOMAIN_1),
       'process.env.JSON_RPC_URL': JSON.stringify(process.env.JSON_RPC_URL)
-    })
+    }),
+    new CopyPlugin([
+      {from: './src/ui/assets/jarvis-logo.png'},
+      {from: './src/_redirects'}
```

Comment:
Are they copied in the same directory as before, so `dist/html/` ?

Reply:
Yes